src/main/java/com/monitorjbl/plugins/AsyncProcessor.java

```diff
@@ -1,32 +1,48 @@
 package com.monitorjbl.plugins;
 
-import com.atlassian.bitbucket.concurrent.BucketProcessor;
+import com.atlassian.bitbucket.util.Version;
+import com.atlassian.bitbucket.util.concurrent.ExecutorUtils;
+import com.atlassian.sal.api.ApplicationProperties;
+import com.atlassian.sal.api.lifecycle.LifecycleAware;
+import com.atlassian.util.concurrent.ThreadFactories;
+import org.slf4j.LoggerFactory;
 
-import javax.annotation.Nonnull;
-import java.io.Serializable;
-import java.util.List;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
-public class AsyncProcessor {
-  private static final String PREFIX = "com.monitorjbl.plugins:pr-harmony:";
-  private final ExecutorService concurrencyService;
+public class AsyncProcessor implements LifecycleAware {
```

Comment:
`LifecycleAware` means Bitbucket Server will automatically trigger `onStart` and `onStop` at appropriate points. For Bitbucket Server 5.8 and older, this allows `onStop` to shutdown the PR Harmony-managed `ExecutorService`.